src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -139,23 +137,29 @@ function RetrieveBannerDynamicAssets() {
         words: data.words.toLocaleString(),
       };
       setStats(localeStats);
+      setTransitionBackground(false);
     }
 
     getStats();
-  }, [telescopeUrl]);
```

Comment:
Why remove this?

Reply:
My bad, didn't mean to. Bringing that back.